news/_posts/2023-10-10-release-20231010.md

```diff
@@ -0,0 +1,43 @@
+---
+kind: article
+title: "Release 20231010 and the Community"
+author: "The OpenRA Team"
+created_at: 2023-10-10 18:00 +0000
+disqus_id: "release-20231010"
+permalink: "/news/release-20231010/"
+---
+
+With two months of successful playtesting behind us, we are pleased to finally announce [Release 20231010]({{ '/download' | relative_url }}), which brings the public release up to date with development changes from November 2022 through to August 2023. The new release contains some long-awaited UI and UX improvements and many polish changes overall. This release serves as a jump off point for many of the larger projects, so do expect great things in the future!
```

Comment:
We should specify which larger projects, just from this its not clear whether you mean mods, tournaments or development projects.

Reply:
I do not want to mention specifics, but as for decreasing the scale of the umbrella I'm not sure which words to use. Perhaps renaming projects to features?

Reply:
`his release serves as a jump off point for many of the larger upcoming features`?